bokehjs/src/lib/models/widgets/button.ts

```diff
@@ -34,6 +34,10 @@ export class Button extends AbstractButton {
   static init_Button(): void {
     this.prototype.default_view = ButtonView
 
+    this.define<Button.Props>(({Number}) => ({
+      clicks: [ Number, 0 ],
```

Comment:
I don't see any `clicks` property on the Python side? What is the use-case for counting clicks?

Reply:
There is none, except for backwards compatibility. Given how broken it is and that noone complained, perhaps it would be better to just remove it.

Reply:
Support for this was removed in Bokeh 2.0 afaik, at least I've had to fake it in Panel since then.

Reply:
Just to explain, the reason I have left `clicks` there is that I didn't want to introduce a new type of event for Panel. I do support the removal in Bokeh though.

Reply:
Right I also thought this disappeared (or was meant to) at 2.0 I would be +1 for removing but if not now then definitely at 3.0